src/core/plugins/core-hud/webview/Hud.vue

```diff
@@ -169,6 +180,22 @@ export default defineComponent({
     margin-bottom: 90px;
 }
 
+.location-placement {
```

Comment:
Unfortunately this is not going to be positioned correctly for very wide screens. We'll need calculations from the client-side to the WebView to determine the exact position of the radar based on what screen resolution they are running.

Example of a minimap on a wider screen setup:
![](https://i.imgur.com/E7m1bsH.png)